src/amo/components/Home/styles.scss

```diff
@@ -109,19 +127,26 @@
 }
 
 .Home-SubjectShelf-list-item .Icon {
-  background-position: center;
   background-repeat: no-repeat;
   background-size: 64px;
   border-radius: $border-radius-default;
   content: '';
   display: block;
   height: 64px;
+  padding-top: 25px;
+  padding-left: 75px;
   margin-bottom: 5px;
   transform: scale(0.8);
   width: 100%;
+  text-align: left;
+
+  @include respond-to(extraSmall) {
+    width: 250%;
```

Comment:
Do you still need this rule? I don't think it was ever getting applied because `extraSmall` is not a valid breakpoint.